math/acl2/Portfile

```diff
@@ -1,57 +1,60 @@
-		a specification language, a modeling language,		\
-		a formal mathematical logic, or a semi-automatic	\
-		theorem prover. Because the meta-language is the same	\
-		as the language (a subset of Common Lisp), it is very	\
-		flexible.
-
-#user_notes	Users who want to use ACL2 for serious work should	\
-#		install the certify variant (sudo port install +certify), \
-#		which will certify (i.e., prove all of the theorems)	\
-#		in the included examples. This can take several hours.
-
-
-distfiles	${name}-${version}${extract.suffix}:sources	\
-		workshops-${version}${extract.suffix}:workshops	\
-		nonstd-${version}${extract.suffix}:nonstd
-
-homepage	http://www.cs.utexas.edu/users/moore/acl2/${shortversion}
-master_sites	${homepage}/distrib/:sources				\
-		${homepage}/distrib/acl2-sources/books:workshops	\
-		${homepage}/distrib/acl2-sources/books:nonstd
-
-checksums	acl2-${version}.tar.gz					\
-                    md5     a2738f4582d14f74664c93dad93ac055		\
-                    sha1    d4052a49a3c2112eeb04cf97570e2caea8df38dd	\
-                    rmd160  167b61cd8a812de804225096f4f3f04b87ba84cb	\
-		workshops-${version}.tar.gz 				\
-                    md5     5ae8123cadfe323c3088a3666112ce07		\
-                    sha1    b3a10d4cd876cb6ccde96920f0103ec0d89f7e2f	\
-                    rmd160  63401f257b8b61564ed660681f82776c762d43da	\
-		nonstd-${version}.tar.gz				\
-                    md5     7de68f04ee468e25b24a230507cdb663		\
-                    sha1    71e1da0cf417ef85f1002279b67227d729822a6e	\
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                acl2
+version             3.5
+set shortversion    v3-5
+categories          math
+maintainers         nomaintainer
+platforms           darwin
+
+description         Applicative Common Lisp / A Computational Logic
+long_description    \
+    ACL2 (Applicative Common Lisp / A Computational \
+    Logic) is the successor to nqthm, the Boyer-Moore \
+    theorem prover. \
+    \
+    ACL2 can be used to automatically or semi-automatically \
+    prove theorems and has been used extensively in real \
+    applications (e.g., proving the correctness of certain \
+    calculations in the floating point unit of the AMD K5 \
+    microprocessor.\
+    \
+    ACL2 is a very large, multipurpose system. \
+    You can use it as a programming language, \
+    a specification language, a modeling language, \
+    a formal mathematical logic, or a semi-automatic \
+    theorem prover. Because the meta-language is the same \
+    as the language (a subset of Common Lisp), it is very \
+    flexible.
+
+# user_notes Users who want to use ACL2 for serious work should
+# install the certify variant (sudo port install +certify),
+# which will certify (i.e., prove all of the theorems)
+# in the included examples. This can take several hours.
```

Comment:
Was this supposed to end up in `notes` at some point in the past?

Reply:
I am not sure exactly what the author wanted to do here.

+certify and +regression takes a lot of time to build (got stuck for me when my mac went to sleep and didn't resume back). These two variants also fails during trace mode for some reason; neverpanic and I were discussing on IRC and couldn't find out the reason.

Something to do with /usr/bin/perl getting replaced by /usr/bin/perl5.18 in sip_workaround folder and throwing not found error.

Most likely we could remove these two variants but some users might want to use +certify. I have tested other variants and also the combinations of them and everything works fine. Atleast the basic functionalities lg2m.

Reply:
Fix it as you prefer and let me know (or merge this yourself). I still think you should add yourself as the maintainer.

Reply:
Ok sure. Thanks :) I'll make the changes regarding the notes and merge it.